csss_site_w3c/events/frosh/tests.py

```diff
@@ -169,5 +169,14 @@ def test_frosh_2022_secret_page(self):
         W3CValidation().validate_page(path="/events/frosh/2022/secret")
 
 
+    def test_frosh_2023_sponsor_page(self):
+        W3CValidation().validate_page(path="/events/frosh/2023/sponsor")
+
+    # the secret page is intended to not be valid html as an easter egg to new froshees
+    @skip("not ready for w3c validation")
```

Comment:
uh, guess I forgot to mention but no new webpages are allowed to use the `@skip` annotation. I use that annotation just on existing pages that I have not had time to fix.

Reply:
I understand, however by design this webpage is supposed to return an error 70% of the time. It's an easter egg.

Reply:
oh, then put that comment in the skip annotation itself